contracts/stellar-interchain-token-service/src/error.rs

```diff
@@ -35,6 +35,7 @@ pub enum ContractError {
     InvalidTokenDecimals = 27,
     ContractPaused = 28,
     InvalidInitialSupply = 29,
+    ConversionError = 30,
```

Comment:
```suggestion
    TokenInvocationError = 30,
```
or something similar, needs to be more readable